### ISSUE TITLE
@font-face css and js error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#194]
```
What steps will reproduce the problem?
1.Using @font-face in an external CSS file.
2.I'm using win XP and IEtester.exe to test older versions of IE6 + 7
3.I'm getting a JS error Line:3 Char:13516 Error: 'style is null or not an
object.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

```

Original issue reported on code.google.com by `mcgarrity@gmail.com` on 4 Sep 2009 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
I'm using the minified version of the script, I've added a simple null check:

for(r=b;r<c;r++){f=d[r];if(f==null || f.style==null) continue; var
s=f.style.cssText;if(f&&(g=s.match(a)))

... and things proceeed... but I guess this is not the best solution...

```

Original comment by `koali...@gmail.com` on 25 Jan 2010 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
I'm experiencing the exact same problem. Your fix seems to prevent the error, 
but it might indeed not be the 
best solution.
```

Original comment by `bartdube...@gmail.com` on 28 Jan 2010 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 6:46
- Changed state: **Fixed**
